src/shared/scrapers/AUS/NSW/index.js

```diff
@@ -1,12 +1,21 @@
+import assert from 'assert';
 import * as parse from '../../../lib/parse.js';
 import * as fetch from '../../../lib/fetch/index.js';
 import maintainers from '../../../lib/maintainers.js';
-import { DeprecatedError } from '../../../lib/errors.js';
 
-/**
- * @param {Cheerio} $row
- */
-const parseRow = $row => parse.number($row.find('td:last-child').text());
+const getKey = rowLabel => {
```

Comment:
I wonder if there's value in some global helper to heuristically parse labels.
May be more effort than its worth though.